packages/api-explorer/package-lock.json

```diff
@@ -11245,41 +11306,31 @@
       }
     },
     "react-jsonschema-form": {
-      "version": "github:domharrington/react-jsonschema-form#fa1cf44205261cf5582e3c12cecf0fd39f548c44",
+      "version": "github:domharrington/react-jsonschema-form#f056a1ed34d2d0841ae897303e9d29b9e8143687",
```

Comment:
I believe this is an outdated version of react-jsonschema-form: https://github.com/domharrington/react-jsonschema-form/commit/f056a1ed34d2d0841ae897303e9d29b9e8143687

We should be using this one instead: https://github.com/domharrington/react-jsonschema-form/commit/fa1cf44205261cf5582e3c12cecf0fd39f548c44